test/dynamo/test_export.py

```diff
@@ -2321,7 +2321,7 @@ def test_exported_graph_serialization(self):
 
         def f(x, y):
             b = x.item()
-            constrain_as_size(b, min=2, max=5)
+            constrain_as_size(b)
             return torch.empty((b, y.shape[0]))
```

Comment:
🤔  Maybe we should automatically constrain as size whenever you pass data dependent values in sizey contexts. This would be consistent with the "special irrefutable assert" construct we've been talking about.

Reply:
(no action needed)

Reply:
Yeah, not bad, we compromise a big of explicit-ness but users get less hassle in most cases.

Reply:
s/big/bit/